schema/instantiate_msg.json

```diff
@@ -21,6 +22,11 @@
     "capital_denom": {
       "type": "string"
     },
+    "capital_per_share": {
+      "type": "integer",
+      "format": "uint64",
```

Comment:
This might be a dumb question but does the `integer` type, `uint64` format allow decimal? I figured it would be a different type....

Reply:
just a 64 bit unsigned integer

Reply:
not sure how likely something would be evenly divisible if something was a decimal (capital per share), for most of our funds this number would be 1 cfigureomni per investment coin

Reply:
Oh I see what you're doing there.....because our denomination would be `cfigureomni` but we display all values in `figureomni/USD' on the FE we use .01 figureomni = 1 fund share which is actually a 1 to 1 capital per share with cfigureomni.... I think 🤔

Reply:
I believe for REIT it would be 10omni ($10) or 1,000cfigureomni ($10) per investment coin